Nef_S2/include/CGAL/Nef_S2/Sphere_segment.h

```diff
@@ -222,6 +223,25 @@ bool operator==(const Sphere_segment<R>& so) const
 bool operator!=(const Sphere_segment<R>& so) const
 { return !operator==(so); }
 
+private:
+
+Point_3 pole() const
```

Comment:
Perhaps this should be renamed `orthogonal_pole` so its consistent with Sphere_circle?